src/main/java/milbot/Storage.java

```diff
@@ -0,0 +1,61 @@
+package milbot;
+import taskclasses.*;
+
+import java.io.*;
```

Comment:
Perhaps specifying the modules imported here to be more concise?

Reply:
Thank you for pointing out, I will update it to be more precise!